server/package.json

```diff
@@ -10,14 +10,16 @@
     "start": "ts-node -r tsconfig-paths/register src/main.ts",
     "start:dev": "concurrently --handle-input \"wait-on dist/main.js && nodemon\" \"tsc -w -p tsconfig.build.json\" ",
     "start:debug": "nodemon --config nodemon-debug.json",
+    "migration:run": "yarn run typeorm migration:run",
```

Comment:
do we need `yarn run` here?

Reply:
yes otherwise it won't find the configuration options (as its specified via the --config param in the typeorm command).

I actually meant to remove this command as we should be using the typeorm command directly